projects/go-tester/src/app/app.guard.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { Router, CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class AppGuard implements CanActivate {
+
+	constructor(private router: Router) { }
```

Comment:
can we look into formatting this file?

Reply:
I suppose, it's totally for testing so... I fixed it though.